.github/workflows/codecov_ats_input_periodic_test_run.yml

```diff
@@ -0,0 +1,82 @@
+name: Periodic tests with --cov-context=test
+# This workflow generates pytest coverage with the flag --cov-context=test
+# This coverage is used as input for Codecov Automated Test Selection (see .github/workflows/codecov_ats.yml)
+# However there's a performance toll in running tests with this flag.
+# So we will be running the test suite every hour in master
+
+on:
+  schedule:
+    # Runs every hour
+    - cron: '0 * * * *'
```

Comment:
this is going to clog up CI if run this often -- this takes from the same pool of workers that all of getsentry open source has

Reply:
I see. What periodicity can that CI take?

Reply:
I'm not sure -- any time this runs it is going to remove capacity from developers so at the parallelism configured here I'm not sure it's going to be acceptable to run at all without impacting developers

Reply:
I just copied the config in `backend.yml` to avoid errors, but I don't really care if this takes a long time to run (let's cap that at "couple hours").

If I reduce the parallelism and increase the schedule time would it be acceptable?

Reply:
you might be able to batch it into the test shuffler?

Reply:
https://github.com/getsentry/sentry/blob/master/.github/workflows/shuffle-tests.yml

Reply:
I understand the concern about developer experience. We certainly don't want to disrupt developers workflows that much.

However I think that running once a week might be too little to get good results, especially in a testing stage in which we are still building confidence in the ATS tool. How can we build confidence in it if we don't give it the data it needs?

So I cut down the parallelism to match the test shuffler and changed the schedule to once a day. It's similar to processing an extra 1.2 PRs a day, and at 1AM UTC.

Reply:
one idea would be to run it in a fork in your own account -- this would use your own worker pool and wouldn't disrupt sentry devs